neonvm-kernel/linux-config-aarch64-6.12.8

```diff
@@ -5006,16 +5121,25 @@ CONFIG_COMMON_CLK_AXG=y
 # CONFIG_COMMON_CLK_AXG_AUDIO is not set
 # CONFIG_COMMON_CLK_A1_PLL is not set
 # CONFIG_COMMON_CLK_A1_PERIPHERALS is not set
+CONFIG_COMMON_CLK_C3_PLL=y
+CONFIG_COMMON_CLK_C3_PERIPHERALS=y
 CONFIG_COMMON_CLK_G12A=y
+CONFIG_COMMON_CLK_S4_PLL=y
+CONFIG_COMMON_CLK_S4_PERIPHERALS=y
 # end of Clock support for Amlogic platforms
 
 CONFIG_ARMADA_AP_CP_HELPER=y
 CONFIG_ARMADA_37XX_CLK=y
 CONFIG_ARMADA_AP806_SYSCON=y
 CONFIG_ARMADA_CP110_SYSCON=y
 CONFIG_QCOM_GDSC=y
-CONFIG_QCOM_RPMCC=y
 CONFIG_COMMON_CLK_QCOM=y
+# CONFIG_CLK_X1E80100_CAMCC is not set
+# CONFIG_CLK_X1E80100_DISPCC is not set
+# CONFIG_CLK_X1E80100_GCC is not set
+# CONFIG_CLK_X1E80100_GPUCC is not set
+# CONFIG_CLK_X1E80100_TCSRCC is not set
+# CONFIG_CLK_QCM2290_GPUCC is not set
```

Comment:
There's a lot of clock drivers here... I think we only _need_ `CONFIG_COMMON_CLK` but we have lots of platform-specific clocks here that are enabled in groups by manufacturer (Amlogic, Qualcomm, etc.).

Probably requires poking around `make menuconfig` a little bit.